### ISSUE TITLE
Data is not loading from json to table

### ISSUE THREAD

**Issue by loveleshsaxena** [wenzhixin/bootstrap-table#1193]
Hi,
I am trying to populate json data in bootstrap table, but the data is not loaded saying "No matching records found". I am using RoR.
Below is the code for html.erb file:
"
<head>

<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">

<link rel="stylesheet" href="bootstrap.min.css">
<link rel="stylesheet" href="bootstrap-table.css">

</head>
<h1>ServiceActions#list</h1>
<h1>Hi there!!!</h1>
<p>Select instances, select service names, and service action from this page:</p>
<%= form_for(:sa_var, url: service_actions_path) do |f| %>
<%
  organization = ""
  project = ""
  if params[:organization].blank?
    organization = "None"
  elsif params[:organization] == "1"
    organization = "MobileSecureProduction"
  elsif params[:organization] == "2"
    organization = "MobileSecurePreprod"
  elsif params[:organization] == "3"
    organization = "MobileSecureDevelopment"
  end
  if params[:project].blank?
    project = "None"
  elsif params[:project] == "1"
    project = "prod"
  elsif params[:project] == "2"
    project = "preprod"
  elsif params[:project] == "3"
    project = "pilot"
  elsif params[:project] == "4"
    project = "devtest"
  elsif params[:project] == "5"
    project = "next"
  end
  display_text = "Organization: #{organization} and Landscape: #{project}"
%>

<%= f.label :organization, value=display_text  %>

<% end %>

<table data-toggle="table" class="table table-bordered table-striped" data-click-to-select="true">
    <thead>
        <tr>
            <th data-checkbox="true">Item ID</th>
            <th>Item Name</th>
            <th>Item Price</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>1</td>
            <td>Item 1</td>
            <td>$1</td>
        </tr>
        <tr>
            <td>2</td>
            <td>Item 2</td>
            <td>$2</td>
        </tr>
    </tbody>
</table>

<table data-toggle="table" data-url="data.json" data-height="299" data-click-to-select="true">
            <thead>
                <tr>
                    <th data-field="id">Item IDs</th>
                    <th data-field="name">Creator</th>
                    <th data-field="price">Created</th>
                </tr>
            </thead>
        </table>
		
<table data-url="data1.json" data-height="299" data-click-to-select="true">
    <thead>
        <tr>
            <th data-field="state" data-checkbox="true"></th>
            <th data-field="id" data-align="right">Item ID</th>
            <th data-field="name" data-align="center">Item Name</th>
            <th data-field="price" data-align="">Item Price</th>
        </tr>
    </thead>
</table>
		

<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>

<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>

<script src="bootstrap-table-zh-CN.js"></script>

<script src="jquery.min.js"></script>
<script src="bootstrap.min.js"></script>
<script src="bootstrap-table.js"></script>
<script type="text/javascript">
$table.bootstrapTable('refresh', {
    url: 'url'
});
</script>
"

Below is the json data:
"
[
{
        "id": 0,
        "name": "Item 0",
        "price": "$0"
    },
    {
        "id": 1,
        "name": "Item 1",
        "price": "$1"
    }
]
"

Request you to please help me on this as it already take my 2 days with no good luck, else my manager will kick me. :(

Lovelesh.

**Comment by djhvscf** [comment]
Provide a fiddle with an example of your JSON response and your table configuration please.. @loveleshsaxena

**Comment by loveleshsaxena** [comment]
Thanks for replying on this. I will provide the fiddle link in few minutes.

Lovelesh.

**Comment by loveleshsaxena** [comment]
Just one quick question: "My aim is to build a table on view, with select check boxes, and data to be loaded from a JSON. This is possible by installing bootstrap gem and installing it through bundle install or I need some extension also for this functionality?"

**Comment by loveleshsaxena** [comment]
Hi,
I am new to Fiddle, hence just copy pasted my html.erb code to there. I don't know to put js or bootstrap css there. Also, in CSS i have pasted my JSON file.

**Comment by djhvscf** [comment]
Use this http://jsfiddle.net/wenyi/e3nk137y/13/light/ as template please.. in the jsfiddle you c<an't use PHP
Just replace the data object with your JSON response and also change the table configuration with you data configuration

**Comment by loveleshsaxena** [comment]
Hi,
My mistake, sorry I was busy in another assignment with higher priority.
The issue was resolved. I need to provide a route config for json file in routes.rb file too. After providing route to that and restarting my server, JSON got loaded.
Thanks every one for looking into this.

Lovelesh.

**Comment by wenzhixin** [comment]
Good news, close this issue.

**Comment by loveleshsaxena** [comment]
Thanks, closed
